### ISSUE TITLE
VS2022 crash when using ChatGPT Turbo Window in 2.4.1

### ISSUE THREAD

**Issue by buhralex** [jeffdapaz/VisualChatGPTStudio#57]
I just installed version 2.4.1 of Visual ChatGPT Studio.
When I open the ChatGPT Turbo Window and click on any previous created chats or write a query in a new chat my VS2022 crashes to the desktop.

Here is the ouput from the Windows Eventviewer Log:
`Anwendung: devenv.exe
Frameworkversion: v4.0.30319
Beschreibung: Der Prozess wurde aufgrund einer unbehandelten Ausnahme beendet.
Ausnahmeinformationen: System.UnauthorizedAccessException
   bei MS.Internal.Text.TextInterface.Native.Util.ConvertHresultToException(Int32)
   bei MS.Internal.Text.TextInterface.FontList.get_Item(UInt32)
   bei MS.Internal.FontFace.PhysicalFontFamily.MapGlyphTypeface(System.Windows.FontStyle, System.Windows.FontWeight, System.Windows.FontStretch, System.Windows.Media.TextFormatting.CharacterBufferRange, System.Globalization.CultureInfo, Int32 ByRef, Int32 ByRef)
   bei MS.Internal.Shaping.TypefaceMap.MapByFontFaceFamily(System.Windows.Media.TextFormatting.CharacterBufferRange, System.Globalization.CultureInfo, System.Globalization.CultureInfo, MS.Internal.FontFace.IFontFamily, System.Windows.FontStyle, System.Windows.FontWeight, System.Windows.FontStretch, MS.Internal.FontFace.PhysicalFontFamily ByRef, Int32 ByRef, MS.Internal.FontFace.IDeviceFont, Boolean, Double, MS.Internal.SpanVector, Int32, Boolean, Int32 ByRef)
   bei MS.Internal.Shaping.TypefaceMap.MapByFontFamily(System.Windows.Media.TextFormatting.CharacterBufferRange, System.Globalization.CultureInfo, System.Globalization.CultureInfo, MS.Internal.FontFace.IFontFamily, MS.Internal.FontCache.CanonicalFontFamilyReference, System.Windows.FontStyle, System.Windows.FontWeight, System.Windows.FontStretch, MS.Internal.FontFace.PhysicalFontFamily ByRef, Int32 ByRef, MS.Internal.FontFace.IDeviceFont, Double, Int32, MS.Internal.SpanVector, Int32, Int32 ByRef)
   bei MS.Internal.Shaping.TypefaceMap.MapOnceByFontFamilyList(System.Windows.Media.TextFormatting.CharacterBufferRange, System.Globalization.CultureInfo, System.Globalization.CultureInfo, System.Windows.Media.FontFamily[], MS.Internal.FontFace.PhysicalFontFamily ByRef, Int32 ByRef, MS.Internal.FontFace.IDeviceFont, Double, Int32, MS.Internal.SpanVector, Int32, Int32 ByRef)
   bei MS.Internal.Shaping.TypefaceMap.MapByFontFamilyList(System.Windows.Media.TextFormatting.CharacterBufferRange, System.Globalization.CultureInfo, System.Globalization.CultureInfo, System.Windows.Media.FontFamily[], MS.Internal.FontFace.PhysicalFontFamily ByRef, Int32 ByRef, MS.Internal.FontFace.IDeviceFont, Double, Int32, MS.Internal.SpanVector, Int32, Int32 ByRef)
   bei MS.Internal.Shaping.TypefaceMap.MapItem(System.Windows.Media.TextFormatting.CharacterBufferRange, System.Globalization.CultureInfo, MS.Internal.Span, MS.Internal.Generic.SpanVector`1<Int32> ByRef, Int32)
   bei MS.Internal.Shaping.TypefaceMap.GetShapeableText(System.Windows.Media.TextFormatting.CharacterBufferReference, Int32, System.Windows.Media.TextFormatting.TextRunProperties, System.Globalization.CultureInfo, Boolean, System.Collections.Generic.IList`1<MS.Internal.TextFormatting.TextShapeableSymbols>, MS.Internal.TextFormatting.IShapeableTextCollector, System.Windows.Media.TextFormattingMode)
   bei MS.Internal.Shaping.GlyphingCache.GetShapeableText(System.Windows.Media.Typeface, System.Windows.Media.TextFormatting.CharacterBufferReference, Int32, System.Windows.Media.TextFormatting.TextRunProperties, System.Globalization.CultureInfo, Boolean, System.Collections.Generic.IList`1<MS.Internal.TextFormatting.TextShapeableSymbols>, MS.Internal.TextFormatting.IShapeableTextCollector, System.Windows.Media.TextFormattingMode)
   bei System.Windows.Media.TextFormatting.TextCharacters.MS.Internal.TextFormatting.ITextSymbols.GetTextShapeableSymbols(MS.Internal.Shaping.GlyphingCache, System.Windows.Media.TextFormatting.CharacterBufferReference, Int32, Boolean, Boolean, System.Globalization.CultureInfo, System.Windows.Media.TextFormatting.TextModifierScope, System.Windows.Media.TextFormattingMode, Boolean)
   bei MS.Internal.TextFormatting.TextStore.CreateTextLSRuns(MS.Internal.TextFormatting.TextRunInfo, System.Collections.Generic.IList`1<System.Windows.Media.TextEffect>, System.Globalization.CultureInfo, Int32, Int32, Int32, System.Windows.Media.TextFormattingMode, Boolean, Int32 ByRef)
   bei MS.Internal.TextFormatting.TextStore.CreateLSRuns(MS.Internal.TextFormatting.TextRunInfo, System.Collections.Generic.IList`1<System.Windows.Media.TextEffect>, System.Globalization.CultureInfo, Int32, Int32, Int32, System.Windows.Media.TextFormattingMode, Boolean, Int32 ByRef, Int32 ByRef)
   bei MS.Internal.TextFormatting.TextStore.CreateLSRunsUniformBidiLevel(MS.Internal.SpanVector, MS.Internal.SpanVector, Int32, Int32, Int32, Int32, System.Windows.Media.TextFormattingMode, Boolean, Int32 ByRef)
   bei MS.Internal.TextFormatting.TextStore.FetchLSRun(Int32, System.Windows.Media.TextFormattingMode, Boolean, MS.Internal.TextFormatting.Plsrun ByRef, Int32 ByRef, Int32 ByRef)
   bei MS.Internal.TextFormatting.LineServicesCallbacks.FetchRunRedefined(IntPtr, Int32, Int32, IntPtr, Char*, Int32, Int32 ByRef, Char* ByRef, Int32 ByRef, Int32 ByRef, MS.Internal.TextFormatting.LsChp ByRef, IntPtr ByRef)

Ausnahmeinformationen: System.UnauthorizedAccessException
   bei MS.Internal.TextFormatting.TextMetrics+FullTextLine.FormatLine(MS.Internal.TextFormatting.FullTextState, Int32, Int32, Int32, Int32, Int32, MS.Internal.TextFormatting.LineFlags, MS.Internal.TextFormatting.FormattedTextSymbols)
   bei MS.Internal.TextFormatting.TextMetrics+FullTextLine..ctor(MS.Internal.TextFormatting.FormatSettings, Int32, Int32, Int32, MS.Internal.TextFormatting.LineFlags)
   bei MS.Internal.TextFormatting.TextFormatterImp.FormatLineInternal(System.Windows.Media.TextFormatting.TextSource, Int32, Int32, Double, System.Windows.Media.TextFormatting.TextParagraphProperties, System.Windows.Media.TextFormatting.TextLineBreak, System.Windows.Media.TextFormatting.TextRunCache)
   bei MS.Internal.TextFormatting.TextFormatterImp.FormatLine(System.Windows.Media.TextFormatting.TextSource, Int32, Double, System.Windows.Media.TextFormatting.TextParagraphProperties, System.Windows.Media.TextFormatting.TextLineBreak)
   bei ICSharpCode.AvalonEdit.Rendering.TextView.CalculateDefaultTextMetrics()
   bei ICSharpCode.AvalonEdit.Rendering.TextView.OnPropertyChanged(System.Windows.DependencyPropertyChangedEventArgs)
   bei System.Windows.DependencyObject.NotifyPropertyChange(System.Windows.DependencyPropertyChangedEventArgs)
   bei System.Windows.DependencyObject.UpdateEffectiveValue(System.Windows.EntryIndex, System.Windows.DependencyProperty, System.Windows.PropertyMetadata, System.Windows.EffectiveValueEntry, System.Windows.EffectiveValueEntry ByRef, Boolean, Boolean, System.Windows.OperationType)
   bei System.Windows.TreeWalkHelper.InvalidateTreeDependentProperty(System.Windows.TreeChangeInfo, System.Windows.DependencyObject, MS.Internal.FrameworkObject ByRef, System.Windows.DependencyProperty, System.Windows.FrameworkPropertyMetadata, System.Windows.Style, System.Windows.Style, System.Windows.ChildRecord ByRef, Boolean, Boolean, Boolean, Boolean)
   bei System.Windows.TreeWalkHelper.InvalidateTreeDependentProperties(System.Windows.TreeChangeInfo, System.Windows.FrameworkElement, System.Windows.FrameworkContentElement, System.Windows.Style, System.Windows.Style, System.Windows.ChildRecord ByRef, Boolean, Boolean, Boolean, Boolean)
   bei System.Windows.FrameworkElement.InvalidateTreeDependentProperties(System.Windows.TreeChangeInfo, Boolean, Boolean)
   bei System.Windows.FrameworkElement.OnAncestorChangedInternal(System.Windows.TreeChangeInfo)
   bei System.Windows.TreeWalkHelper.OnAncestorChanged(System.Windows.DependencyObject, System.Windows.TreeChangeInfo, Boolean)
   bei System.Windows.DescendentsWalker`1[[System.Windows.TreeChangeInfo, PresentationFramework, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35]].StartWalk(System.Windows.DependencyObject, Boolean)
   bei MS.Internal.PrePostDescendentsWalker`1[[System.Windows.TreeChangeInfo, PresentationFramework, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35]].StartWalk(System.Windows.DependencyObject, Boolean)
   bei System.Windows.TreeWalkHelper.InvalidateOnTreeChange(System.Windows.FrameworkElement, System.Windows.FrameworkContentElement, System.Windows.DependencyObject, Boolean)
   bei System.Windows.FrameworkElement.OnVisualParentChanged(System.Windows.DependencyObject)
   bei System.Windows.Media.Visual.FireOnVisualParentChanged(System.Windows.DependencyObject)
   bei System.Windows.Media.Visual.AddVisualChild(System.Windows.Media.Visual)
   bei System.Windows.FrameworkElement.set_TemplateChild(System.Windows.UIElement)
   bei System.Windows.Controls.ContentPresenter+UseContentTemplate.BuildVisualTree(System.Windows.FrameworkElement)
   bei System.Windows.StyleHelper.ApplyTemplateContent(System.Windows.UncommonField`1<System.Collections.Specialized.HybridDictionary[]>, System.Windows.DependencyObject, System.Windows.FrameworkElementFactory, Int32, System.Collections.Specialized.HybridDictionary, System.Windows.FrameworkTemplate)
   bei System.Windows.FrameworkTemplate.ApplyTemplateContent(System.Windows.UncommonField`1<System.Collections.Specialized.HybridDictionary[]>, System.Windows.FrameworkElement)
   bei System.Windows.FrameworkElement.ApplyTemplate()
   bei System.Windows.FrameworkElement.MeasureCore(System.Windows.Size)
   bei System.Windows.UIElement.Measure(System.Windows.Size)
   bei System.Windows.Controls.DockPanel.MeasureOverride(System.Windows.Size)
   bei System.Windows.FrameworkElement.MeasureCore(System.Windows.Size)
   bei System.Windows.UIElement.Measure(System.Windows.Size)
   bei System.Windows.Controls.Control.MeasureOverride(System.Windows.Size)
   bei System.Windows.FrameworkElement.MeasureCore(System.Windows.Size)
   bei System.Windows.UIElement.Measure(System.Windows.Size)
   bei MS.Internal.Helper.MeasureElementWithSingleChild(System.Windows.UIElement, System.Windows.Size)
   bei System.Windows.Controls.ScrollContentPresenter.MeasureOverride(System.Windows.Size)
   bei System.Windows.FrameworkElement.MeasureCore(System.Windows.Size)
   bei System.Windows.UIElement.Measure(System.Windows.Size)
   bei System.Windows.Controls.Grid.MeasureCell(Int32, Boolean)
   bei System.Windows.Controls.Grid.MeasureCellsGroup(Int32, System.Windows.Size, Boolean, Boolean, Boolean ByRef)
   bei System.Windows.Controls.Grid.MeasureOverride(System.Windows.Size)
   bei System.Windows.FrameworkElement.MeasureCore(System.Windows.Size)
   bei System.Windows.UIElement.Measure(System.Windows.Size)
   bei System.Windows.Controls.ScrollViewer.MeasureOverride(System.Windows.Size)
   bei System.Windows.FrameworkElement.MeasureCore(System.Windows.Size)
   bei System.Windows.UIElement.Measure(System.Windows.Size)
   bei System.Windows.Controls.Border.MeasureOverride(System.Windows.Size)
   bei System.Windows.FrameworkElement.MeasureCore(System.Windows.Size)
   bei System.Windows.UIElement.Measure(System.Windows.Size)
   bei System.Windows.Controls.Control.MeasureOverride(System.Windows.Size)
   bei System.Windows.FrameworkElement.MeasureCore(System.Windows.Size)
   bei System.Windows.UIElement.Measure(System.Windows.Size)
   bei System.Windows.Controls.Grid.MeasureCell(Int32, Boolean)
   bei System.Windows.Controls.Grid.MeasureCellsGroup(Int32, System.Windows.Size, Boolean, Boolean, Boolean ByRef)
   bei System.Windows.Controls.Grid.MeasureOverride(System.Windows.Size)
   bei System.Windows.FrameworkElement.MeasureCore(System.Windows.Size)
   bei System.Windows.UIElement.Measure(System.Windows.Size)
   bei MS.Internal.Helper.MeasureElementWithSingleChild(System.Windows.UIElement, System.Windows.Size)
   bei System.Windows.Controls.ContentPresenter.MeasureOverride(System.Windows.Size)
   bei System.Windows.FrameworkElement.MeasureCore(System.Windows.Size)
   bei System.Windows.UIElement.Measure(System.Windows.Size)
   bei System.Windows.Controls.StackPanel.StackMeasureHelper(System.Windows.Controls.IStackMeasure, System.Windows.Controls.IStackMeasureScrollData, System.Windows.Size)
   bei System.Windows.Controls.StackPanel.MeasureOverride(System.Windows.Size)
   bei System.Windows.FrameworkElement.MeasureCore(System.Windows.Size)
   bei System.Windows.UIElement.Measure(System.Windows.Size)
   bei MS.Internal.Helper.MeasureElementWithSingleChild(System.Windows.UIElement, System.Windows.Size)
   bei System.Windows.Controls.ItemsPresenter.MeasureOverride(System.Windows.Size)
   bei System.Windows.FrameworkElement.MeasureCore(System.Windows.Size)
   bei System.Windows.UIElement.Measure(System.Windows.Size)
   bei System.Windows.Controls.Border.MeasureOverride(System.Windows.Size)
   bei System.Windows.FrameworkElement.MeasureCore(System.Windows.Size)
   bei System.Windows.UIElement.Measure(System.Windows.Size)
   bei System.Windows.Controls.Control.MeasureOverride(System.Windows.Size)
   bei System.Windows.FrameworkElement.MeasureCore(System.Windows.Size)
   bei System.Windows.UIElement.Measure(System.Windows.Size)
   bei MS.Internal.Helper.MeasureElementWithSingleChild(System.Windows.UIElement, System.Windows.Size)
   bei System.Windows.Controls.ScrollContentPresenter.MeasureOverride(System.Windows.Size)
   bei System.Windows.FrameworkElement.MeasureCore(System.Windows.Size)
   bei System.Windows.UIElement.Measure(System.Windows.Size)
   bei System.Windows.Controls.Grid.MeasureCell(Int32, Boolean)
   bei System.Windows.Controls.Grid.MeasureCellsGroup(Int32, System.Windows.Size, Boolean, Boolean, Boolean ByRef)
   bei System.Windows.Controls.Grid.MeasureOverride(System.Windows.Size)
   bei System.Windows.FrameworkElement.MeasureCore(System.Windows.Size)
   bei System.Windows.UIElement.Measure(System.Windows.Size)
   bei System.Windows.Controls.ScrollViewer.MeasureOverride(System.Windows.Size)
   bei System.Windows.FrameworkElement.MeasureCore(System.Windows.Size)
   bei System.Windows.UIElement.Measure(System.Windows.Size)
   bei System.Windows.Controls.Grid.MeasureCell(Int32, Boolean)
   bei System.Windows.Controls.Grid.MeasureCellsGroup(Int32, System.Windows.Size, Boolean, Boolean, Boolean ByRef)
   bei System.Windows.Controls.Grid.MeasureOverride(System.Windows.Size)
   bei System.Windows.FrameworkElement.MeasureCore(System.Windows.Size)
   bei System.Windows.UIElement.Measure(System.Windows.Size)
   bei System.Windows.Controls.Grid.MeasureCell(Int32, Boolean)
   bei System.Windows.Controls.Grid.MeasureCellsGroup(Int32, System.Windows.Size, Boolean, Boolean, Boolean ByRef)
   bei System.Windows.Controls.Grid.MeasureOverride(System.Windows.Size)
   bei System.Windows.FrameworkElement.MeasureCore(System.Windows.Size)
   bei System.Windows.UIElement.Measure(System.Windows.Size)
   bei MS.Internal.Helper.MeasureElementWithSingleChild(System.Windows.UIElement, System.Windows.Size)
   bei System.Windows.Controls.ContentPresenter.MeasureOverride(System.Windows.Size)
   bei System.Windows.FrameworkElement.MeasureCore(System.Windows.Size)
   bei System.Windows.UIElement.Measure(System.Windows.Size)
   bei System.Windows.Controls.Border.MeasureOverride(System.Windows.Size)
   bei System.Windows.FrameworkElement.MeasureCore(System.Windows.Size)
   bei System.Windows.UIElement.Measure(System.Windows.Size)
   bei System.Windows.Controls.Control.MeasureOverride(System.Windows.Size)
   bei System.Windows.FrameworkElement.MeasureCore(System.Windows.Size)
   bei System.Windows.UIElement.Measure(System.Windows.Size)
   bei MS.Internal.Helper.MeasureElementWithSingleChild(System.Windows.UIElement, System.Windows.Size)
   bei System.Windows.Controls.ContentPresenter.MeasureOverride(System.Windows.Size)
   bei System.Windows.FrameworkElement.MeasureCore(System.Windows.Size)
   bei System.Windows.UIElement.Measure(System.Windows.Size)
   bei System.Windows.ContextLayoutManager.UpdateLayout()
   bei System.Windows.Controls.TabItem.OnPreviewGotKeyboardFocus(System.Windows.Input.KeyboardFocusChangedEventArgs)
   bei System.Windows.RoutedEventArgs.InvokeHandler(System.Delegate, System.Object)
   bei System.Windows.RoutedEventHandlerInfo.InvokeHandler(System.Object, System.Windows.RoutedEventArgs)
   bei System.Windows.EventRoute.InvokeHandlersImpl(System.Object, System.Windows.RoutedEventArgs, Boolean)
   bei System.Windows.UIElement.RaiseEventImpl(System.Windows.DependencyObject, System.Windows.RoutedEventArgs)
   bei System.Windows.UIElement.RaiseTrustedEvent(System.Windows.RoutedEventArgs)
   bei System.Windows.Input.InputManager.ProcessStagingArea()
   bei System.Windows.Input.InputManager.ProcessInput(System.Windows.Input.InputEventArgs)
   bei System.Windows.Input.KeyboardDevice.TryChangeFocus(System.Windows.DependencyObject, System.Windows.Input.IKeyboardInputProvider, Boolean, Boolean, Boolean)
   bei System.Windows.Input.KeyboardDevice.Focus(System.Windows.DependencyObject, Boolean, Boolean, Boolean)
   bei System.Windows.Input.KeyboardDevice.Focus(System.Windows.IInputElement)
   bei System.Windows.UIElement.Focus()
   bei System.Windows.Controls.TabItem.SetFocus()
   bei System.Windows.Controls.TabControl.OnSelectionChanged(System.Windows.Controls.SelectionChangedEventArgs)
   bei System.Windows.Controls.Primitives.Selector+SelectionChanger.End()
   bei System.Windows.Controls.Primitives.Selector+SelectionChanger.SelectJustThisItem(ItemInfo, Boolean)
   bei System.Windows.Controls.Primitives.Selector.OnSelectedItemChanged(System.Windows.DependencyObject, System.Windows.DependencyPropertyChangedEventArgs)
   bei System.Windows.DependencyObject.OnPropertyChanged(System.Windows.DependencyPropertyChangedEventArgs)
   bei System.Windows.FrameworkElement.OnPropertyChanged(System.Windows.DependencyPropertyChangedEventArgs)
   bei System.Windows.DependencyObject.NotifyPropertyChange(System.Windows.DependencyPropertyChangedEventArgs)
   bei System.Windows.DependencyObject.UpdateEffectiveValue(System.Windows.EntryIndex, System.Windows.DependencyProperty, System.Windows.PropertyMetadata, System.Windows.EffectiveValueEntry, System.Windows.EffectiveValueEntry ByRef, Boolean, Boolean, System.Windows.OperationType)
   bei System.Windows.DependencyObject.SetValueCommon(System.Windows.DependencyProperty, System.Object, System.Windows.PropertyMetadata, Boolean, Boolean, System.Windows.OperationType, Boolean)
   bei System.Windows.DependencyObject.SetValue(System.Windows.DependencyProperty, System.Object)
   bei JeffPires.VisualChatGPTStudio.ToolWindows.Turbo.TerminalWindowTurboControl.OpenTab(JeffPires.VisualChatGPTStudio.ToolWindows.Turbo.ChatUserControlsItem)
   bei System.Windows.RoutedEventArgs.InvokeHandler(System.Delegate, System.Object)
   bei System.Windows.RoutedEventHandlerInfo.InvokeHandler(System.Object, System.Windows.RoutedEventArgs)
   bei System.Windows.EventRoute.InvokeHandlersImpl(System.Object, System.Windows.RoutedEventArgs, Boolean)
   bei System.Windows.UIElement.RaiseEventImpl(System.Windows.DependencyObject, System.Windows.RoutedEventArgs)
   bei System.Windows.Controls.Control.HandleDoubleClick(System.Object, System.Windows.Input.MouseButtonEventArgs)
   bei System.Windows.RoutedEventArgs.InvokeHandler(System.Delegate, System.Object)
   bei System.Windows.RoutedEventHandlerInfo.InvokeHandler(System.Object, System.Windows.RoutedEventArgs)
   bei System.Windows.EventRoute.InvokeHandlersImpl(System.Object, System.Windows.RoutedEventArgs, Boolean)
   bei System.Windows.UIElement.ReRaiseEventAs(System.Windows.DependencyObject, System.Windows.RoutedEventArgs, System.Windows.RoutedEvent)
   bei System.Windows.UIElement.OnMouseDownThunk(System.Object, System.Windows.Input.MouseButtonEventArgs)
   bei System.Windows.RoutedEventArgs.InvokeHandler(System.Delegate, System.Object)
   bei System.Windows.RoutedEventHandlerInfo.InvokeHandler(System.Object, System.Windows.RoutedEventArgs)
   bei System.Windows.EventRoute.InvokeHandlersImpl(System.Object, System.Windows.RoutedEventArgs, Boolean)
   bei System.Windows.UIElement.RaiseEventImpl(System.Windows.DependencyObject, System.Windows.RoutedEventArgs)
   bei System.Windows.UIElement.RaiseTrustedEvent(System.Windows.RoutedEventArgs)
   bei System.Windows.Input.InputManager.ProcessStagingArea()
   bei System.Windows.Input.InputManager.ProcessInput(System.Windows.Input.InputEventArgs)
   bei System.Windows.Input.InputProviderSite.ReportInput(System.Windows.Input.InputReport)
   bei System.Windows.Interop.HwndMouseInputProvider.ReportInput(IntPtr, System.Windows.Input.InputMode, Int32, System.Windows.Input.RawMouseActions, Int32, Int32, Int32)
   bei System.Windows.Interop.HwndMouseInputProvider.FilterMessage(IntPtr, MS.Internal.Interop.WindowMessage, IntPtr, IntPtr, Boolean ByRef)
   bei System.Windows.Interop.HwndSource.InputFilterMessage(IntPtr, Int32, IntPtr, IntPtr, Boolean ByRef)
   bei MS.Win32.HwndWrapper.WndProc(IntPtr, Int32, IntPtr, IntPtr, Boolean ByRef)
   bei MS.Win32.HwndSubclass.DispatcherCallbackOperation(System.Object)
   bei System.Windows.Threading.ExceptionWrapper.InternalRealCall(System.Delegate, System.Object, Int32)
   bei System.Windows.Threading.ExceptionWrapper.TryCatchWhen(System.Object, System.Delegate, System.Object, Int32, System.Delegate)
   bei System.Windows.Threading.Dispatcher.LegacyInvokeImpl(System.Windows.Threading.DispatcherPriority, System.TimeSpan, System.Delegate, System.Object, Int32)
   bei MS.Win32.HwndSubclass.SubclassWndProc(IntPtr, Int32, IntPtr, IntPtr)

`

**Comment by jeffdapaz** [comment]
Hi @buhralex,

this is a very stranger bug. What I saw It's look like for somehow the VS couldn't access a specific font family.

Any chance that you changed some default VS font? If so, can you change back to the default font only to check if is this the error cause?

You can too run the VS as admin to check if this can resolve the issue?

By the way, I made a litle search and found this related bugs:

https://developercommunity.visualstudio.com/t/vs2020-preview-41-crashing-on-lanuch/1546980
https://developercommunity.visualstudio.com/t/cascadia-codemono-crashes-vs-2022-editor/1580696
https://developercommunity.visualstudio.com/t/Visual-Studio-Crashes-when-switching-the/1496960

Can you read then and check if the workarounds that they gave can resolve the extension issue?

**Comment by buhralex** [comment]
Thanks @jeffdapaz,

yeah I saw that too and remembered when I updated VS a few days ago I've read something about the default font for VS changed in this release. Upon start VS told me that it was not able to load the default font and just displayed an empty window. So I was forced to change the default font in the VS settings.

I will check the links tomorrow and see if I can somehow fix this.

What's really strange is that I can use the other Visual ChatGPT Studio windows without an issue. Only the Turbo window is not working.

Thanks for taking your time. Will report back
